tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -91,12 +91,12 @@ bfd/test_bfd.py::test_bfd_echo_mode:
 
 bfd/test_bfd.py::test_bfd_scale:
   skip:
-    reason: "Test is not verified for cisco-8111 and cisco-8122 yet.
-             and not supported for platforms other than Nvidia 4600c/4700/5600. Skipping the test / KVM do not support bfd test"
+    reason: "Test is not verified for cisco-8111 and cisco-8122 yet. \
+             and not supported for platforms other than Nvidia 4280/4600c/4700/5600. Skipping the test / KVM do not support bfd test"
     conditions_logical_operator: or
     conditions:
       - "platform in ['x86_64-8111_32eh_o-r0', 'x86_64-8122_64eh_o-r0', 'x86_64-8122_64ehf_o-r0']"
-      - "platform not in ['x86_64-mlnx_msn4600c-r0', 'x86_64-mlnx_msn4700-r0', 'x86_64-nvidia_sn5600-r0', 'x86_64-8102_64h_o-r0', 'x86_64-8101_32fh_o-r0']"
+      - "platform not in ['x86_64-mlnx_msn4600c-r0', 'x86_64-mlnx_msn4700-r0', 'x86_64-nvidia_sn5600-r0', 'x86_64-8102_64h_o-r0', 'x86_64-8101_32fh_o-r0', 'x86_64-nvidia_sn4280-r0', 'x86_64-8102_28fh_dpu_o-r0']"
```

Comment:
Same as above.

Reply:
@bingwang-ms this was added by https://github.com/sonic-net/sonic-mgmt/pull/14955. Adding @yutongzhang-microsoft to comment. @wangxin for viz.

Reply:
Same as above